CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Show comments multiline @rodfersou
```

Comment:
Also, it's a fix more than a feature, right? Then it goes to changes. We use semantic versioning, so if it's truly a feature=> minor increase, if it's a fix/patch=> patch increase.